README.md

```diff
@@ -874,6 +874,7 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 * [theimpossibleastronaut/configster](https://github.com/theimpossibleastronaut/configster) [[configster](https://crates.io/crates/configster)] — Rust library for parsing configuration files [<img src="https://api.travis-ci.com/theimpossibleastronaut/configster.svg?branch=trunk">](https://travis-ci.com/theimpossibleastronaut/configster)
 * [Kixunil/configure_me](https://github.com/Kixunil/configure_me) [[configure_me](https://crates.io/crates/configure_me)] — library for processing application configuration easily
 * [andoriyu/uclicious](https://github.com/andoriyu/uclicious) [[uclicious](https://crates.io/crates/uclicious)] — [libUCL](https://github.com/vstakhov/libucl) based feature-rich configuration library. [![CircleCI](https://circleci.com/gh/vstakhov/libucl.svg?style=svg)](https://circleci.com/gh/vstakhov/libucl)
+* [FlashSystems/justconfig](https://crates.io/crates/justconfig) - Easily extendable, layered configuration library that introduces no additional dependencies into your project.
```

Comment:
Please add a link to the repository as per the other links above this

Reply:
I adhered to the template from the [Contributing](https://github.com/rust-unofficial/awesome-rust/blob/master/CONTRIBUTING.md) document. Should I make a pull request to change that as well?

Reply:
Argh. Yes please. We've got a bit of a mix of entries (which one of these years needs tidying), and the "link to your repository, plus crate link if you've got it" is the best one.